### DISCUSSION TITLE
How to build System.Text.Json?

### DISCUSSION THREAD

**Discussion by Maximys** [dotnet/runtime#55497] [Q&A] [answered]
I want to build and run tests of [System.Text.Json](https://github.com/dotnet/runtime/tree/main/src/libraries/System.Text.Json), but currently I get next error:

> 6>C:\runtime\src\libraries\System.Text.Json\gen\System.Text.Json.SourceGeneration.csproj : error MSB4057: The target "GetTargetPath" does not exist in the project.

Can anybody explain me what's wrong?
[BuildLog-minimal.txt](https://github.com/dotnet/runtime/files/6799411/BuildLog-minimal.txt)
[BuildLog-Diagnostic](https://drive.google.com/file/d/1xFyBmbqnr6VeWxpMymGJcMAk9ATvwwh5/view?usp=sharing)

**Comment by gfoidl** [comment]
Does https://github.com/dotnet/runtime/tree/main/docs/workflow/building/libraries help you?

**Reply by Maximys** [reply]
I think, my problem linked with [Issue-53434](https://github.com/dotnet/runtime/issues/53434).
Currently I trying to build System.Text.Json by provided instruction, but I think, it will not help me.

**Comment by ViktorHofer** [comment, marked as answer]
This was a regression that existed for 1-2 days in the source code. @Maximys can you please try building Systme.Text.Json again?

**Reply by Maximys** [reply]
@ViktorHofer , thank you so much, I had just compiled System.Text.Json without any problems
